### PR TITLE
Minesweeper Script

### PR DESCRIPTION
**Pull request by Ratnesh4193**
# Description

Added script for MINESWEEPER game 
Fixes: #738 

Capture

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

  Yes

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

## Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [ ] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

@kaustubhgupta plz review

### CODE REVIEW

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]
@Ratnesh4193 code is working fine, but add a few features like when the game is over, add an option to restart the game and exit the game

**Comment by Ratnesh4193** [comment]
added restart game feature plz review @santushtisharma10

**Comment by Ratnesh4193** [comment]
@kaustubhgupta plz review

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]
@Ratnesh4193 when the game restarts, make sure that the score is zero instead of adding the previous score.

**Comment by kaustubhgupta** [comment]
> @kaustubhgupta plz review

Let @santushtisharma10 review it first, after that I will review

**Comment by kaustubhgupta** [comment]
@Ratnesh4193 updates on this?

**Comment by Ratnesh4193** [comment]
I have fixed the bug. Now plz review @santushtisharma10 @kaustubhgupta

File: MineSweeper/readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Minesweeper
+
+This is a python script MINESWEEPER game 
+
+$--- tkinter
```
Comment (kaustubhgupta) [inline MineSweeper/readme.md:5]:
```suggestion
## Modules Used
- tkinter
```

File: MineSweeper/readme.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+$--- random (for randomness in game)
```
Comment (kaustubhgupta) [inline MineSweeper/readme.md:7]:
```suggestion
-  random (for randomness in game)
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by santushtisharma10** [comment]
@Ratnesh4193 intitialize the score to zero when the game restart

**Comment by Ratnesh4193** [comment]
> @Ratnesh4193 initialize the score to zero when the game restart

Done

**Review by santushtisharma10 — APPROVED** [review]
@Ratnesh4193  💯

**Comment by santushtisharma10** [comment]
@Kushal997-das

**Review by antrikshmisri — APPROVED** [review]

**Review by kaustubhgupta — APPROVED** [review]
@Ratnesh4193 💯

**Review by avinashkranjan — APPROVED** [review]
LGTM..💯

**Pull request merged**